src/main/java/org/fundacionjala/coding/david/movies/Movie.java

```diff
@@ -0,0 +1,65 @@
+package org.fundacionjala.coding.david.movies;
+
+/**
+ * Handle movie retail.
+ */
+public abstract class Movie {
+    private String title;
+
+    /**
+     * Movie constructor.
+     *
+     * @param title movie title
+     */
+    public Movie(final String title) {
+        this.title = title;
+    }
+
+    /**
+     * Get extra points from specific movie type.
+     *
+     * @return int points
+     */
+    public abstract int getPoints();
```

Comment:
Do you think the setPoints and getPoint should be abstract methods?
or maybe you can create an abstract method 'calculateFrequentRenterPoints'